### PR TITLE
Updated PGF Meeting time [Wait on confirmation]

### PR DESCRIPTION
**Pull request by jbusecke**
PR to update the PGF working group time. 

- [x] Still waiting on confirmation from @keewis (and maybe others that want to attend?)
- [x] @maxrjones can we use a numfocus conferencing link to not make this dependent on anyone attending?
    - [x] Change link in the calendar invite (@jbusecke has access)
    - [x] Update link here

### CODE REVIEW

File: src/data/meetings.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ export const WorkingGroups = [
   {
     name: 'Pangeo Forge',
     description: 'Coordinating development on Pangeo Forge.',
-    schedule: 'Every other Monday at 2pm ET',
+    schedule: 'Every other Wednesday at 2pm ET',
```
Comment (keewis) [inline src/data/meetings.js:56]: the time is fine with me, but we might want to choose the offset week in a way that doesn't conflict with the [zarr community meetings](https://zarr.dev/community-calls/)

File: src/data/meetings.js [hunk 2]
```diff
@@ -53,7 +53,7 @@ export const WorkingGroups = [
...
     conferencing: 'https://whereby.com/pangeo',
```
Comment (maxrjones) [inline src/data/meetings.js:57]:
```suggestion
    conferencing: 'https://numfocus-org.zoom.us/j/81511924663?pwd=3rLhw0R2jqUmua9hyp2sCg8rdtQXAY.1',
```

File: src/data/meetings.js (hunk 1 above)
Comment (jbusecke) [reply src/data/meetings.js:56]: We plan to start on Oct 23, so that should be good?

**Comment by jbusecke** [comment]
@maxrjones thx! Can anyone join that link or do we need to assign multiple hosts?

**Comment by jbusecke** [comment]
Also I believe this is good to merge! We will double check the [starting week](https://github.com/pangeo-data/pangeo.io/pull/35#discussion_r1792493039) for the calendar invite!

**Comment by maxrjones** [comment]
> @maxrjones thx! Can anyone join that link or do we need to assign multiple hosts?

I think anyone can join, the only limitation to my knowledge is that others can't record

**Comment by jbusecke** [comment]
I think that is not required for PGF at this stage. Thanks for setting this up. I think from our end this can get merged!

**Pull request merged**